modules/installation-generate-aws-user-infra-ignition.adoc

```diff
@@ -61,6 +62,49 @@ Because you create and manage the worker machines yourself, you do not need
 to initialize these machines.
 endif::[]
 
+ifeval::["{context}" == "installing-gcp-user-infra"]
+. Remove the Kubernetes manifest files that define the worker machines:
++
+----
+rm -f openshift/99_openshift-cluster-api_worker-machineset-*.yaml
+----
++
+Because you create and manage the worker machines yourself, you do not need
+to initialize these machines.
+
+. Currently, emptying the compute pools makes control plane machines schedulable. However, due to a link:https://github.com/kubernetes/kubernetes/issues/65618[Kubernetes limitation], router Pods running on control plane machines will not be reachable by the ingress load balancer.
++
+Ensure the `mastersSchedulable` parameter is set to `false` in the
```

Comment:
This is also covered in the in-flight #17016.  Seems like we'd want to land that first and then just use the same module wording for AWS and GCP?  I think the overlap between the AWS and GCP wording would be almost the whole module.

Reply:
@kalexand-rh LMK if you want to merge that first.